packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -379,12 +377,12 @@ export function TransferPanel() {
             return
           }
 
-          const { allowance } = await getL1TokenData(
-            walletAddress,
-            selectedToken.address,
-            l1.provider,
-            l2.provider
-          )
+          const { allowance } = await getL1TokenData({
+            account: walletAddress,
+            erc20L1Address: selectedToken.address,
+            l1Provider: l1Provider,
+            l2Provider: l2Provider
```

Comment:
can use the object property shorthand here
```typescript
{
  ...
  l1Provider,
  l2Provider
}
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer

Reply:
To avoid this in the future, we can turn on eslint [rule](https://eslint.org/docs/latest/rules/object-shorthand) for it:

Reply:
Ticket created: https://www.notion.so/arbitrum/Add-object-shorthand-eslint-rule-264342979cef4153b3744150a4762492